### PR TITLE
Resume reasoning similarity score

### PR DESCRIPTION
**Pull request by Anjalipolasani**
1 .Added the similarity score reasoning using the model paraphrase-MiniLM-L6-v2 
2. Tested around approximately 10 resumes with job description found good  results 

if the similarity score is greater than 60:

It indicates a high similarity between the resume and the job description.
The resume has skills, experiences, and qualifications that closely match the job, meaning the candidate is well-suited for the position

If the similarity score is between 30 and 60:

It indicates moderate similarity.
The resume contains some relevant skills or experiences, but it may not fully meet all the job requirements. There's overlap, but it's not a perfect match.

If the similarity score is 30 or less:

It indicates low similarity.
The resume does not align well with the job description, and the candidate likely lacks important qualifications or experiences needed for the role.

This is a .py file for this model I have uploaded .ipnyb in this branch

### CODE REVIEW

File: Resume_reasoning_similarity_score (1).ipynb [hunk 1]
```diff
@@ -0,0 +1,171 @@
+"""Resume_reasoning_similarity_score.ipynb
```
Comment (zeshengchen) [inline Resume_reasoning_similarity_score (1).ipynb]: This file seems a .py file, instead of .ipynb file!

Comment (Anjalipolasani) [reply Resume_reasoning_similarity_score (1).ipynb]: professor, it is a file rename issue I am trying to upload .ipnyb file

**Review by zeshengchen — APPROVED** [review]
I am going to approve this pull request.

But I do not think it is a good idea to put the file under the root of the project.

**Review by anushkabhatcode — CHANGES_REQUESTED** [review]
Anjali, as professor already indicated change file location to models folder under backend since that is the folder we will be using to deploy backend


I have uploaded the files .py extension in the models folder which  can integrate through backend

**Review by pktpaulie — APPROVED** [review]

**Review by Kavyan17 — APPROVED** [review]

**Review by KarthikBalajiB — APPROVED** [review]

**Review by anushkabhatcode — APPROVED** [review]

**Pull request merged**